app/views/profile.jsx

```diff
@@ -174,15 +174,15 @@ var Profile = React.createClass({
               <Col md={3}>
                 <Select className='filterTag'
                   clearable={false}
-                  placeholder='标签'
-                  value={this.state.query.tag == '' ? '标签' : this.state.query.tag}
+                  placeholder='Label'
+                  value={this.state.query.tag == '' ? 'Label' : this.state.query.tag}
                   disabled={this.state.LMT.tags.length ? false : true}
                   options={this.state.LMT.tags}
                   onChange={this.filterTag}
                   />
               </Col>
               <Col md={2}>
-                <Button bsStyle='primary' onClick={this.reset}>重置</Button>
+                <Button bsStyle='primary' onClick={this.reset}>"Reset filter"</Button>
```

Comment:
Maybe take a look at this one. Not sure if the quotation marks are required or not.